Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure('2') do |config|
+  config.vm.box = 'ubuntu/bionic64'
+  config.vm.box_check_update = false
+  config.vm.host_name = 'sumologic-kubernetes-collection-dev.vagrantup.com'
+
+  config.vm.network :forwarded_port, guest: 18080, host: 18080
```

Comment:
Port forwards are not needed with `:private_network`, those will just work out of the box. Please remove.